index.php

```diff
@@ -3,40 +3,43 @@
 <head>
     <title>Kate Starks</title>
     <link rel='stylesheet' type='text/css' href='css/normalize.css'/>
-    <link href="https://fonts.googleapis.com/css?family=Quicksand:300,500" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Nova+Square|Open+Sans" rel="stylesheet">
     <link rel='stylesheet' type='text/css' href='css/styles.css'/>
 </head>
 
 <body>
-    <main class="hero">
+    <main>
+        <div class="hero">
```

Comment:
I've used this because I could only position the text at the bottom of the viewport through absolute positioning - if the 'main' container is removed from the document flow it changes the positioning of the subsequent sections